app/views/layouts/_header.html.erb

```diff
@@ -11,11 +11,23 @@
       </div>
       <div class="pull-right auto-width-right">
         <ul class="top-details menu-beta l-inline">
-          <li><a href="#"><i class="fa fa-user"></i>
-            <%= t "header.account" %></a>
-          </li>
-          <li><%= link_to t("header.log_in"), login_path %></li>
-          <li><%= link_to t("header.sign_up"), signup_path %></li>
+          <% if logged_in? %>
+            <li>
+              <a href="javascript:void(0)">
+                <%= "#{t('header.hello')} #{current_user.full_name}" %>
```

Comment:
đọc chưa kỹ rồi
http://guides.rubyonrails.org/i18n.html#passing-variables-to-translations